modules/ROOT/pages/errors/gql-errors/22N38.adoc

```diff
@@ -1,4 +1,12 @@
 = 22N38
 
 == Status description
-error: data exception - invalid function argument. Invalid argument to the function `{ $fun }`.
+error: data exception - invalid function argument. Invalid argument to the function `{ <<fun>> }`.
```

Comment:
Of some reason, this parameter is "value" in the codebase, but "fun" in docs and the spreadsheet. I think "fun" makes more sense so considering changing it in the codebase